src/librustdoc/config.rs

```diff
@@ -383,27 +383,28 @@ impl Options {
             }
         }
 
+        let edition = matches.opt_str("edition").unwrap_or("2015".to_string());
```

Comment:
We tend to prefer `.unwrap_or_else`. ;)

Reply:
This was existing code that i moved for this PR, but i can fix it.